docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -248,6 +248,33 @@
               message: 'Clock on {{ $labels.instance }} is not synchronising. Ensure NTP is configured on this host.',
             },
           },
+          {
+            alert: 'NodeRAIDDegraded',
+            expr: |||
+              'node_md_disks_required - ignoring (state) (node_md_disks{state="active"}) > 0'
+            ||| % $._config,
+            'for': '15m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'RAID Array is degraded',
+              message: 'RAID array '{{ $labels.device }}' is in degraded state due to one or more disks failures. Number of spare drives is insufficient to fix issue automatically.',
```

Comment:
Perhaps add the instance? E.g.: "RAID array '{{ $labels.device }}' on {{ $labels.instance }} is in degraded state…"?

Reply:
And let's s/message/description/ . The remaining `message` annotations above are currently being changed to `description`, see #1764 .